src/Entity/InformationProductType.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+
+namespace App\Entity;
+
+use App\Repository\InformationProductTypeRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=InformationProductTypeRepository::class)
+ */
+class InformationProductType extends Entity
+{
+    use DescriptorTypeTrait;
```

Comment:
Is there additional refactor that will use this trait? Otherwise, why use a trait once?

Reply:
We thought we will use it for Digital Resource type which also would have only 'id' and 'description' attributes